src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -393,4 +436,27 @@ public void setCachePath(String cachePath) {
         this.cachePath = cachePath;
     }
 
+	public String getOfflineResourcesBaseUrl() {
+		return offlineResourcesBaseUrl;
+	}
+
+	public void setOfflineResourcesBaseUrl(String offlineResourcesBaseUrl) {
+		this.offlineResourcesBaseUrl = offlineResourcesBaseUrl;
+	}
+
+	public List<DataSourceEnum> getMsgOriginsQueue() {
+		return msgOriginsQueue;
+	}
+
+	public void setMsgOriginsQueue(List<DataSourceEnum> msgOriginsQueue) {
```

Comment:
I don't see how to use this function 'setMsgOriginsQueue' and how to create the parameter 'msgOriginsQueue'.

Reply:
This function is not used, I can remove it if you like.
See VIPCfg.initialize: 
```// Remote VIP resources takes priority over offline resources
        // so add DataSourceEnum.VIP first to msgOriginsQueue
        if (prop.containsKey("vipServer")) {
            this.vipServer = prop.getString("vipServer");
            this.msgOriginsQueue.add(DataSourceEnum.VIP);
        }
        if (prop.containsKey("offlineResourcesBaseUrl")) {
        	this.offlineResourcesBaseUrl = prop.getString("offlineResourcesBaseUrl");
        	this.msgOriginsQueue.add(DataSourceEnum.Bundle);
        }
```